pkgs/tools/networking/gmrender-resurrect/default.nix

```diff
@@ -39,8 +31,8 @@ in
     meta = with lib; {
       description = "Resource efficient UPnP/DLNA renderer, optimal for Raspberry Pi, CuBox or a general MediaServer";
       homepage = "https://github.com/hzeller/gmrender-resurrect";
-      license = licenses.gpl2;
+      license = licenses.gpl2Plus;
       platforms = platforms.linux;
-      maintainers = with maintainers; [ koral ashkitten ];
```

Comment:
ping @ashkitten

Reply:
@ashkitten requested to be removed from the maintainers list earlier in this review.

Reply:
Didn't read that. 👍🏼